### ISSUE TITLE
Add/wrap performance testing/extended debug commands that are Sauce specific

### ISSUE THREAD

**Issue by christian-bromann** [saucelabs/simple_sauce#52]
The DevTools team has created a bunch of WebDriver extensions over the last year that might worth being added to the driver in various of languages. In WebdriverIO I already added these commands:

https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-protocols/protocols/saucelabs.json

The document should be self describing but it would be cool if users could call those commands directly instead of using the JS Executor workaround.

**Comment by nadvolod** [comment]
@christian-bromann awesome suggestion 👍 I started something similar a few months back but we decided to hold off for the Alpha and Beta versions. But likely will be released for v1.0.
Are you thinking something like [this](https://github.com/saucelabs/simple_sauce/blob/0865f071ab48cc2ab1f199ff5d79c4191187564e/java/src/test/java/com/saucelabs/simplesauce/unit/SauceApiTest.java#L22)
or [this](https://github.com/saucelabs/simple_sauce/blob/0865f071ab48cc2ab1f199ff5d79c4191187564e/java/src/test/java/com/saucelabs/simplesauce/unit/SauceApiTest.java#L29)? Just to make sure we're on the same page.

**Comment by christian-bromann** [comment]
> Are you thinking something like [this](https://github.com/saucelabs/simple_sauce/blob/0865f071ab48cc2ab1f199ff5d79c4191187564e/java/src/test/java/com/saucelabs/simplesauce/unit/SauceApiTest.java#L22)

No, what I thinking of here is to extend the Selenium client with these sauce specific commands so that someone can call e.g. `driver.throttleNetwork('Regular #F')`

**Comment by nadvolod** [comment]
@christian-bromann cool, so basically performance-related stuff as I can tell from the doc?

**Comment by christian-bromann** [comment]
@nadvolod yes, these commands are mainly performance/extended debugging related
